tiledb/sm/query/result_space_tile.h

```diff
@@ -85,19 +82,77 @@ struct ResultSpaceTile {
   /** Default move-assign operator. */
   ResultSpaceTile& operator=(ResultSpaceTile&& result_space_tile) = default;
 
+  /** Returns the fragment domains. */
+  const std::vector<std::pair<unsigned, std::reference_wrapper<const NDRange>>>&
```

Comment:
Hmm, we follow the same in so many places. Whenever I see a const reference I always think that if the object disappears I may lose that info. Whenever I want to prevent that I return a new object that does not depend on the members of a class instance.